### PR TITLE
Add pro and enterprise main scripts

### PR DESCRIPTION
**Pull request by kminehart**

### CODE REVIEW

File: scripts/drone_publish_main.sh [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
+  --distro=windows/amd64 \
+  --distro=darwin/amd64 \
+  --checksum \
   --build-id=${DRONE_BUILD_NUMBER} \
   --grafana-dir=${GRAFANA_DIR} \
   --github-token=${GITHUB_TOKEN} \
   --go-version=${GO_VERSION} \
-  --destination=${DESTINATION}/${DRONE_BUILD_EVENT} \
-  --gcp-service-account-key-base64=${GCP_KEY_BASE64}
+  --version=${DRONE_TAG} \
```
Comment (dsotirakis) [inline scripts/drone_publish_main.sh:25]: Hm, I think `DRONE_TAG` is not available to `main` builds? Maybe we need the same thing that goes in the `--version` arg here?

**Review by dsotirakis — COMMENTED** [review]
This PR doesn't cover the `-` vs `~` issue, yeah?

**Comment by kminehart** [comment]
> This PR doesn't cover the - vs ~ issue, yeah?

It does! That's handled in the `move_scripts.go` command at the end.

File: scripts/drone_publish_main.sh (hunk 1 above)
Comment (kminehart) [reply scripts/drone_publish_main.sh:25]: yeah you're right, good catch. I need to change that.

**Review by dsotirakis — APPROVED** [review]
👀

**Pull request merged**